easybuild/easyconfigs/c/cpu_features/cpu_features-0.6.0-foss-2020b.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'CMakeMake'
+
+name = 'cpu_features'
+version = '0.6.0'
+
+homepage = 'https://github.com/google/cpu_features'
+description = """A cross-platform C library to retrieve CPU features (such as available instructions) at runtime."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
This should be changed to `GCCcore` (which implies renaming the easyconfig file and adding `binutils` as a build dependency as well), to ensure this can be used as a dependency for non-`foss` toolchains too.

Reply:
done